wcfapp/README.md

```diff
@@ -0,0 +1,53 @@
+# WCF Docker Container Samples
+These samples demonstrate how to dockerize WCF services, either IIS-hosted or self-hosted. A simple "hello world" service contract is used in all samples for both HTTP and net.tcp transport bindings.
+
+## Prepare Your Environment
+After cloning or downloading this repository, build the solution in the `wcfapp` folder either in Visual Studio or using command line `msbuild wcfapp.sln` in a VS console window. You will need environment and software below installed to complete these samples.
+
+* [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) (or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server)) with **'Containers'** feature enabled.
+*  [Docker for Windows](https://docs.docker.com/docker-for-windows/) with **'Switch to Windows Containers...'** by right clicking on the Docker whale icon in system tray
+* [Visual Studio 2017](https://www.visualstudio.com/vs/)
+
+## Run WCF Service in Container
+WCF service is supported on .NET Framework, which can run in Windows Server Core based cotnainers.
+
+### Build a Container with IIS-hosted WCF Service
+Project `WcfServiceWebApp` is created from 'WCF Service Application' template in Visual Studio. A [Dockerfile](WcfServiceWebApp/Dockerfile) is added to the project. We use `microsoft/wcf` image as the base image, which has both http and net.tcp protocols enabled in IIS and exposes ports 80 (for http) and 808 (for net.tcp) for the container. We use tag `4.6.2` for .NET Framework 4.6.2 in this example, but you can change it to use other tags (eg. `microsoft/wcf:4.7`) for WCF images with different versions of .NET Framework.
+
+Run commands below to build the container image with name `wcfservice:iis-hosted` and start an instance of it named `service`. Docker parameter `-d` will run the container in detached mode.
+```
+C:\wcfapp\WcfServiceWebApp>docker build -t wcfservice:iis-hosted .
+C:\wcfapp\WcfServiceWebApp>docker run -d --name service wcfservice:iis-hosted
```

Comment:
It's intentional to use different names. This way I don't have to explain later that you have to kill the first container instance because we are using the same name 😄. But I do see your point about "consistency", so I changed to use myservice1, myservice2, etc. Hopefully, it's consistent... in a way 😄.